<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <widget name="__qt_fake_top_level">
  <widget class="QPushButton" name="PatientsButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Patients</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SortButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sort</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DashboardButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dashboard</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AssignButton">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>280</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Assign</string>
   </property>
  </widget>
  <widget class="QListWidget" name="SortSpecialityList">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>341</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="RequestPageLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Requests Page</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="IncomingRequestWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>931</width>
     <height>111</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Patient Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Gender</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Birtday</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Speciality</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Doctor Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Visit Date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Request Reason</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="ShowAllButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Show All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RequestsButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Requests</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RejectButton">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>280</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reject</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="DoctorTree">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>330</y>
     <width>351</width>
     <height>231</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Doctor Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Status</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="SortBySpecialityLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select to Sort by Speciality:</string>
   </property>
  </widget>
  <widget class="QLabel" name="IncomingPatientRequestlabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Incoming Patient Request:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DoctorsButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Doctors</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LogoutButton">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>10</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
 </widget>
 <resources/>
</ui>
